wellregistry/registry/models.py

```diff
@@ -95,6 +97,7 @@ def __str__(self):
 
 class UnitsLookup(models.Model):
     """Model definition for the units_dim table, lookup only"""
+    unit_id = models.IntegerField(unique=True)
```

Comment:
It's interesting that I added this too but how to add a default. But if this works, I just assume that we don't add a default